app/views/messages/_messages_for_library.html.erb

```diff
@@ -1,24 +1,48 @@
 <div class="messages">
-<% messages = @messages.select { |m| m.library == library_code && m.request_type == request_type } %>
-<% messages.each do |message| %>
+
+  <% messages = @messages.select { |m| m.library == library_code && m.request_type == request_type } %>
+
+  <% if library_code == 'SAL' %>
+    <% library_code = "SAL1&2" %>
```

Comment:
What about this: https://github.com/sul-dlss/sul-requests/blob/master/app/models/library_location.rb#L41-L43

Which you can see an example of in the `Messages` class here https://github.com/sul-dlss/sul-requests/blob/d9abd56e7a508fc98682c11b1cb9c9f3956e68f5/app/models/message.rb#L15-L17

(which it appears that you've actually duplicated that code a few times in the view, which I'll add comments about separately)